tests/test_importers.py

```diff
@@ -115,11 +127,12 @@ def _class(manager):
 
 
 class TestImporters(TestBaseImporters):
+    CARD_IMPORTERS = ['encryptr',]
```

Comment:
Could you use lower case for this variable name. Because it is only used in `test_importers_card`. It is useless to have it as class parameter.